R/FilteredDataset-utils.R

```diff
@@ -40,6 +40,7 @@
 #'   shinyApp(ui, server)
 #' }
 #'
+#' @examplesIf requireNamespace("MultiAssayExperiment")
```

Comment:
nice 👍 good call to split and make conditional. Much better than big if inside.

Reply:
wow @examplesIf is a powerful feature. great use with `requireNamespace`